### ISSUE TITLE
[Bug] Thermal Innovation Fluxbore CoFH Core Enchantment Anomalies

### ISSUE THREAD

**Issue by CammyKun** [CoFH/Feedback#1018]
Currently running the latest version of each of the CoFH mods, and I attempted to add Insight III to a fluxbore, as shown here
![javaw_2018-04-08_02-38-51](https://user-images.githubusercontent.com/36810356/38465173-06667ece-3ad6-11e8-904e-cc43301f3c2e.png)
Unfortunately the only level of Insight I seem to be able to apply is Insight I. This issue does not happen with any of the other CoFH enchantments,
Likewise, in JEI when I check applicable enchantments for the Fluxbore, it's actually showing me Smelting II-IV, which shouldn't even exist. I'm uncertain if this is due to me having too many mods adding enchantments (OpenBlocks, Quark, Vanilla Tech, and Random Things), or if this is simply a bug with the new release. If I come across anything else involving the topic I'll post it in here.

**Comment by KingLemming** [comment]
Can you try in a new world?

I'm afraid I know what's going on here and if I am correct, this is a vanilla problem.

**Comment by KingLemming** [comment]
Sorry, should have been clear.

Insight can't be applied to Fluxbores atm. That's a bug, and is now fixed in dev.

What's happening with JEI (and why you *think* you can put Insight I on) is that enchants are being remapped when you join the server because of existing ones.

However, JEI doesn't remap the books. I've discussed this with @mezz before and there's not a super awesome way to handle it I believe.

So basically you're seeing a JEI glitch due to enchantment ID remapping on the backend. And the Insight thing is a bug, fixed in next release. :)

**Comment by mezz** [comment]
I think the JEI issue should be resolved in the latest version by this change:  https://github.com/mezz/JustEnoughItems/commit/ff2d9de6581b55f1288a660ead52cd60d7d6fe6b
